### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by jcrist** [jcrist/venv-pack#6]

**Comment by jcrist** [comment]
Indeed. I don't currently have time to add windows support, help is definitely wanted here if you'd like to make a PR.

**Comment by offa** [comment]
With #17 it works fine on windows. Where does the `nt` directory come from? Current python version on windows (at least 3.6 / 3.7) do not create it in their venvs.

**Comment by jcrist** [comment]
> Where does the `nt` directory come from?

We need to add `cmd` scripts mirroring the posix scripts available in `venv-pack/scripts/common` to `venv-pack/scripts/nt`. Without them, there will be no `activate` script available in the packed environment.

**Comment by gipeto** [comment]
I made an attempt to solve this issue in PR #21

**Comment by usercnmd** [comment]
I'm still experiencing this issue in `venv-pack` 0.2.0 on Windows 11 with Python 3.10.x (please replace `x` with your specific version from `python --version`).  The `scripts\nt` directory is missing after installing `venv-pack`, causing a `FileNotFoundError` during packing.

Manually creating an empty `nt` directory in `site-packages\venv_pack\scripts` works as a temporary workaround.

I've confirmed I'm only using Python 3.10, and I've tried reinstalling `venv-pack` and creating new virtual environments. It seems like this might be a regression bug, as it was previously reported fixed. Could you please investigate? Thanks!

**Comment by usercnmd** [comment]
I'm still experiencing this issue in `venv-pack` 0.2.0 on Windows 11 with Python 3.10.11.  The `scripts\nt` directory is missing after installing `venv-pack`, causing a `FileNotFoundError` during packing.

Manually creating an empty `nt` directory in `site-packages\venv_pack\scripts` works as a temporary workaround.

I've confirmed I'm only using Python 3.10, and I've tried reinstalling `venv-pack` and creating new virtual environments. It seems like this might be a regression bug, as it was previously reported fixed. Could you please investigate? Thanks!
